composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.3.7",
         "christian-riesen/base32": "~1.0",
-        "simplesoftwareio/simple-qrcode" : "1.3.*"
+        "simplesoftwareio/simple-qrcode" : "1.4.*"
```

Comment:
Please change this into `^1.3`. This would all all versions from 1.3 till, not including 2.0.